shaptools/hana.py

```diff
@@ -174,13 +178,14 @@ def install(cls, software_path, conf_file, root_user, password):
             conf_file (str): Path to the configuration file
             root_user (str): Root user name
             password (str): Root user password
+            remote_host (str, opt): Remote host name where to command will be executed
```

Comment:
this can be improved.

Remote host where executing commands . Imho currenlty this doesn't sound really eng

Reply:
Update to:
```
remote_host (str, opt): Remote host where the command will be executed
```
What do you think?

Reply:
Yop